### DISCUSSION TITLE
Don't work in Oracle Cloud VM

### DISCUSSION THREAD

**Discussion by RilusMahmud** [wg-easy/wg-easy#1283] [Q&A]
I've tried everything, but the client can not access the internet.
Anyone know anything about oracle cloud issues with wireguard?

**Comment by divadwong68** [comment]
Can you list the things you tried?  Your everything might not be everything.  Do the basics like open cloud firewall, etc.

**Reply by RilusMahmud** [reply]
Okay, firstly I get the docker compose file and made the necessary changes and get it up and running, then head to the oracle cloud console and opened the necessary port with udp 51820 and tcp 51821, then head to the web view in 51821 port with my public ip for the vm, then created a client and connected with my phone with the qr code, then connected with the vpn, but then there is no internet showing up! 
can you help me with if I’m missing something?

also, i’ve tried in in azure cloud and also aws cloud, all working fine. But in oracle it’s not working.

**Reply by offizium-berndstorath** [reply]
Did you also allow the ports in your firewall on the system? with oracle linux its firewalld

**Comment by divadwong68** [comment]
Looks like you did all the steps and it should work.  Maybe just verify that the hostname to wireguard is correct and double check the ports for 51820

**Reply by RilusMahmud** [reply]
Yes, I've done almost everything, and yet couldn't use the vpn as expected. Double and triple checked the ports and also tried with different ports rather then the default ports, all the same! 
Maybe it's something have to do with the oracle cloud policies or something like that. 
I'll have to look into it more deeply I guess!

**Comment by divadwong68** [comment]
the only other gotcha I can think of is if you're using Cloudflare DNS and you have the proxy enabled.  ping your dns name and make sure it's going to the correct IP.
